website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -88,20 +111,191 @@ metrics: [String!]!
+	inputMeasures: [MetricInputMeasure!]!
+	numerator: MetricInput
+	denominator: MetricInput
+	expr: String
+	window: MetricTimeWindow
+	grainToDate: TimeGranularity
+	metrics: [MetricInput!]
+}
+```
+
+**Where filter**
+Update to what's decided
+
+**Dimension Types**
+
+```graphql
+Dimension {
+	name: String!
+	qualifiedName: String!
+	description: String
+	type: DimensionType!
+	typeParams: DimensionTypeParams
+	isPartition: Boolean!
+	expr: String
+}
+```
+```
+DimensionType = [CATEGORICAL, TIME]
+```
+
+### Examples 
+
+**Query two metrics grouped by time**
+
+```graphql
+mutation {
+  createQuery(
+    environmentId: <env_id>
+    metrics: ["food_order_amount", "order_gross_profit"]
+    groupBy: ["metric_time"] - TODO update with syntax for object
+  ) {
+    queryId
+  }
+}
+```
+
+**Query with a time grain** 
+
+```graphql
+mutation {
+  createQuery(
+    environmentId: <env_id>
+    metrics: ["food_order_amount", "order_gross_profit"]
+    groupBy: ["metric_time__month"] - TODO update with syntax for object
+  ) {
+    queryId
+  }
+}
+```
+
+**Query with a time grain** 
+
+```graphql
+mutation {
+  createQuery(
+    environmentId: <env_id>
+    metrics: ["food_order_amount", "order_gross_profit"]
+    groupBy: ["metric_time__month"] - TODO update with syntax for object
+  ) {
+    queryId
+  }
+}
+```
+
+**Query with a categorical Dimension**
+
+```graphql
+mutation {
+  createQuery(
+    environmentId: <env_id>
+    metrics: ["food_order_amount", "order_gross_profit"]
+    groupBy: ["metric_time__month", "customer__customer_type")] - TODO update with syntax for object
+  ) {
+    queryId
+  }
+}
+```
+
+**Query with a Where Filter** 
+
+ The where filter takes a list argument (or a string for a single input). Depending on the object you are filtering on, there are a couple of parameters:
+ 
+ - `Dimension()` - This is used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+- `Entity()` - used for entities like primary and foreign keys - `Entity('order_id')`
+
+Note: If you prefer more strongly typed `where` clause, you can optionally use `TimeDimension` to separate out categorical dimensions from time ones. The `TimeDimension` input takes the time dimension name and also requires granularity - an example is `TimeDimension('metric_time', 'MONTH')`.
+
+```graphql
+mutation {
+	createQuery(
+		metrics:["order_total"]
+		groupBy:["customer__customer_type"]
+		where:["{{ Dimension('customer__customer_type') }} = 'new'", "{{ TimeDimension('metric_time', 'MONTH') }} > '2022-10-01'"] - TODO UPDATE THIS
+	) {
+	queryId
+	}
+}
+```
+
+
+**Query with Limit and Order By** 
+
+```graphql
+mutation {
+  createQuery(
+    environmentId: <env_id>
+    metrics: ["food_order_amount", "order_gross_profit"]
+    groupBy: ["metric_time__month", "customer__customer_type)] - TODO update with syntax for object
+    limit: TODO
```

Comment:
@WilliamDee  do you have an example of what limit/order looks like?

Reply:
Both order and limit will work exactly the same as the JDBC one, so something like
- `limit: Optional[int]` ie., `limit: 10` for limiting the result to 10 rows
- `order: Optional[List[str]]` ie., `order=['-order_gross_profit', "metric_time"]` the `-` is for ordering in DESC

Reply:
@WilliamDee  thanks! wait so does order need the order name before the colon as well as the order outside of the list bracket?

I would assume it would be 

order: ["-revenue"] rather than order: order=['revenue']